submissions/js/task_5_6.MD

```diff
@@ -1,5 +1,27 @@
-| 👩‍🎓 Student Name | 🆔 Enrollment Number | 🔗 Task 5 URL | 🔗 Task 6 URL | 🐱 GitHub Repository URL |
-|---|---|---|---|---|
+| Name               | Enrollment Number | Task 5       | Task 6       | Github Repository |
+|--------------------|-------------------|--------------|--------------|-------------------|
+| ayush              | 2203031080028     |              |              |                   |
+| pranav faladu      | 2303031087022     | https://inquisitive-croquembouche-acec27.netlify.app/multiplayerdicegame | https://inquisitive-croquembouche-acec27.netlify.app/converter |https://github.com/PranavFaladu/JSprojects |
+| Rishabh            | 2203031080114     |              |              |                   |
+| Vekariya Aryan     | 2203031080125     |              |              |                   |
+| Rajdeepsinh        | 2203031080069     |              |              |                   |
+| vishang lad        | 2203031080035     |https://jsprojects3.pages.dev/dice | https://jsprojects3.pages.dev/converter | https://github.com/vishangl/JSprojects                  |
```

Comment:
Sync the changes in the current repo. Your PR is overidding other mentee's contributions!